src/components/tuiter/index.js

```diff
@@ -33,6 +33,7 @@ function Tuiter () {
               <Route path="/tuiter/:uid" element={<Home/>}/>
               <Route path="/home" element={<Home/>}/>
               <Route path="/home/:uid" element={<Home/>}/>
+              <Route path="/explore" element={<Explore/>}/>
               <Route path="/explore/*" element={<Explore/>}/>
```

Comment:
Looks like this is here twice now?

Reply:
This is on purpose, as /explore will be the basic "explore" page, while /explore/* will allow us to alter the explore url to have explore/BBQLife to fill in BBQLife tuits and have the value in the search bar.